packages/instantsearch.js/src/connectors/breadcrumb/connectBreadcrumb.ts

```diff
@@ -233,7 +238,42 @@ const connectBreadcrumb: BreadcrumbConnector = function connectBreadcrumb(
         };
       },
 
-      getWidgetSearchParameters(searchParameters) {
+      getWidgetUiState(uiState, { searchParameters }) {
```

Comment:
somewhat hidden feature here: breadcrumb is now usable without hierarchicalMenu

Reply:
That's pretty cool! Do we actually test that?